miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -339,14 +340,11 @@ private static DetailedSample toDetailedSample(DetailedSampleDto from) {
   }
 
   /**
-   * Extracts parent details from the DTO, according to three possible cases:
+   * Extracts parent details from the DTO, according to these possible cases:
    * 
    * <ol>
    * <li>parent ID is provided. This implies that the parent exists, so no other parent information will be required</li>
-   * <li>identity information and parentSampleClassId are provided. This implies that a tissue parent should be created, and that the
-   * identity may or may not yet exist</li>
-   * <li>identity information is provided, but no parentSampleClassId. This implies that the direct parent of this sample will be an
-   * identity, which may or may not yet exist</li>
+   * <li>there is no parent ID provided, but there is the information to reconstruct the hierarchy</li>
```

Comment:
What information? This documentation is much more useful if it's specific since it's easy to miss something when writing JSON